Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/packs.yml

```diff
@@ -14,8 +14,13 @@
   - type: SpaceGarbage
   - type: Storage
     capacity: 6
+    whitelist:
+      tags:
+      - Cigarette
+      - Cigar
+      - Matchstick
   - type: Item
-    size: 6
+    size: 5
```

Comment:
Please don't make size less than capacity, the item size situation is already bad enough.

This also can't fit basic stuff like pens or joints or whatever?

Reply:
Currently, its whitelsited to cigs, cigars, and matches. Tahts why I made size less capacity.
Ill make it 5\5 no whitelist then